cls/SourceControl/Git/Production.cls

```diff
@@ -477,11 +504,21 @@ ClassMethod RemoveItem(internalName, noFolders As %Boolean = 0) As %Status
         if 'isProdSettings {
             set production = ##class(Ens.Config.Production).%OpenId(productionName,,.sc)
             quit:$$$ISERR(sc)
-            set configItem = ##class(Ens.Config.Production).OpenItemByConfigName(productionName_"||"_itemName_"|"_itemClassName,.sc)
-            quit:$$$ISERR(sc)
-            do production.RemoveItem(configItem)
+             set configItem = ##class(Ens.Config.Production).OpenItemByConfigName(productionName_"||"_itemName_"|"_itemClassName,.sc)
+		    
+		    // only remove config item if it still exists and if item was opened ok
+		    if $$$ISERR(sc) {
+			    if '(sc [ "ErrConfigItemNotFound") {
```

Comment:
There's a more structured way to do this but on further consideration I don't care, this is more readable anyway!